Makefile

```diff
@@ -39,9 +39,24 @@ jsdocs:
 	#Output the final documentation link
 	echo "To read the documentation, click here : file://"${DIR}"/jsdoc/ownCloud.html"
 
+browser:
+	sed -i "s/require('fs/require('fs-web/g" owncloud/*.js
```

Comment:
`sed` sounds a bit barbaric here. Is there any best practice here that could be used ?